### PR TITLE
Update fixture generation and add a zstd metadata fixture

### PR DESCRIPTION
**Pull request by dralley**
Closes https://github.com/pulp/pulp-fixtures/issues/245
Closes https://github.com/pulp/pulp-fixtures/issues/242

### CODE REVIEW

**Comment by dralley** [comment]
@quba42 Any idea why the Debian portion is failing to build?  I assume this is the line: https://github.com/pulp/pulp-fixtures/actions/runs/6885272791/job/18729164889?pr=244#step:5:1941

I'm not sure if this has to do with swapping out the previous key though, because as far as I can tell I fixed it up in all the correct places?

File: .github/workflows/issue_checker.yaml [hunk 1]
```diff
@@ -5,13 +5,11 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```
Comment (mdellweg) [inline .github/workflows/issue_checker.yaml:8]: :+1:

File: common/GPG-KEY-pulp-qe
Comment (mdellweg) [inline common/GPG-KEY-pulp-qe:1]: What called for this change?
I believe some CI is using the key directly from here. So changing it's name is probably a bad idea.
As is changing the key user-id.

File: Containerfile [hunk 2]
```diff
@@ -2,18 +2,21 @@
...
               ostree \
               patch \
+              python3-pip \
               python3-jinja2-cli \
               rpm-build \
               rpm-sign \
               rsync
 
+# the default createrepo_c provided by Fedora has legacy hashes disabled, the one
+# on PyPI does not (because we need it)
```
Comment (mdellweg) [inline Containerfile:18]: What happens if that package would be upgraded also? Want to add a version constraint?

Comment (dralley) [reply Containerfile:18]: Nah.  It's actually good for something like that to make a bit of noise early on.

But also, the Python package is basically our responsibility in the first place, so it's unlikely to change out from under us.

File: common/GPG-KEY-pulp-qe
Comment (dralley) [reply common/GPG-KEY-pulp-qe:1]: The key is ancient and uses algorithms that recent OS' consider legacy, and refuse to do anything with.

Comment (dralley) [reply common/GPG-KEY-pulp-qe:1]: https://github.com/pulp/pulp-fixtures/issues/245

**Review by mdellweg — APPROVED** [review]

**Review by dkliban — APPROVED** [review]

**Pull request merged**